packages/twenty-server/src/workspace/workspace-query-builder/interfaces/workspace-query-builder-options.interface.ts

```diff
@@ -4,7 +4,7 @@ import { FieldMetadataInterface } from 'src/metadata/field-metadata/interfaces/f
 import { ObjectMetadataInterface } from 'src/metadata/field-metadata/interfaces/object-metadata.interface';
 
 export interface WorkspaceQueryBuilderOptions {
-  targetTableName: string;
+  objectMetadataItem: ObjectMetadataInterface;
```

Comment:
@charlesBochet TBC, but I think we've access to the `objectMetadata` from the field entity as the relation is retrieve.